generators/openapi-client/prompts.js

```diff
@@ -160,7 +160,7 @@ function askActionType() {
         this.props = props;
 
         if (props.jhipsterEndpoint !== undefined) {
-            return fetchSwaggerResources(props.jhipsterEndpoint).then(availableDocs => {
+            fetchSwaggerResources(props.jhipsterEndpoint).then(availableDocs => {
```

Comment:
This return is required.
```suggestion
            return fetchSwaggerResources(props.jhipsterEndpoint).then(availableDocs => {
```